rust/monovertex/src/transformer.rs

```diff
@@ -1,61 +1,160 @@
-use crate::error::Result;
-use crate::message::Message;
+use std::collections::HashMap;
+
+use crate::config::config;
+use crate::error::{Error, Result};
+use crate::message::{Message, Offset};
```

Comment:
I suppose this `Message` will be moved to a separate package that will be shared by both MonoVertex and Pipeline.